views/tagShowMovies.ejs

```diff
@@ -0,0 +1,4 @@
+<h1>Movies tagged with <%= tag.tag %></h1>
```

Comment:
Here, we can list associated favorites since we've passed them in from the backend:
```
<h1>Movies tagged with "<%= tag.tag %>"</h1>
<% tag.favorites.forEach(function(favorite) { %>
	<h3><%= favorite.title %></h3>
<% }); %>
```